frontend/lib/src/components/elements/CodeBlock/StreamlitSyntaxHighlighter.tsx

```diff
@@ -74,7 +76,7 @@ export default function StreamlitSyntaxHighlighter({
 
   return (
     <StyledCodeBlock className="stCode" data-testid="stCode">
-      <StyledPre>
+      <StyledPre style={height ? { height: `${height}px` } : undefined}>
```

Comment:
Applying `style` to a styled-comonent feels a bit weird. I think `height` should be moved to `StyledPre` and provided as a parameter.

Reply:
Done